packages/vscode-dependencies-validation/src/editorChanges.ts

```diff
@@ -48,8 +50,12 @@ function executeRefreshDiagnosticsOnEditorChange(
   diagnosticCollection: DiagnosticCollection
 ) {
   return (editor: TextEditor | undefined) => {
-    if (editor) {
-      void refreshDiagnostics(editor.document.uri, diagnosticCollection);
+    if (editor && shouldBeChecked(editor.document.uri.path)) {
+      {
+        const uri = editor.document.uri;
```

Comment:
same as above

Reply:
same as the reply above.
`editor` may be undefined so we may not have a `uri` here...